### ISSUE TITLE
Option to default to query mode

### ISSUE THREAD

**Issue by maxharlow** [szermatt/visual-replace#12]
Thanks for a great package! Would be great if there was an option so query mode (ie. calling `query-replace` instead of `replace-string`) could be enabled by default.

**Comment by szermatt** [comment]
The following should do what you want. It'll work with the stable version and the latest version from git:

```
(add-hook 'visual-replace-minibuffer-mode-hook
          (defun my-visual-replace-setup ()
            (visual-replace-toggle-query)))
```

Unfortunately, this won't work with the version currently in unstable melpa (20241106.934) because there was a bug, since fixed (in 530904e) That fix will be in the unstable melpa version tomorrow.

**Comment by szermatt** [comment]
With `use-package`, there's also a more elegant way of doing that:

```
(use-package visual-replace
  [...]
  :hook ((visual-replace-minibuffer-mode . visual-replace-toggle-query))
```
